<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEventForm</class>
 <widget class="QDialog" name="EditEventForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QCheckBox" name="chk_isPrivado">
     <property name="text">
      <string>Evento privado</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reunion_group">
     <property name="title">
      <string>Cita con</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="combo_cliente"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../terra.qrc">
          <normaloff>:/Icons/PNG/search.png</normaloff>:/Icons/PNG/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="especialidad_group">
     <property name="title">
      <string>Especialidad</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="combo_espec"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="departamento_group">
     <property name="title">
      <string>Departamento</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="combo_dep"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_tituloEvento">
     <property name="placeholderText">
      <string>Titulo...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txt_asunto"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Desde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="time_start">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>H:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hasta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="time_end">
       <property name="displayFormat">
        <string>H:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_getColor">
       <property name="styleSheet">
        <string notr="true">background-color: red</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../terra.qrc">
         <normaloff>:/Icons/PNG/undo.png</normaloff>:/Icons/PNG/undo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_guardar">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="../terra.qrc">
         <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>reunion_group</zorder>
  <zorder>especialidad_group</zorder>
  <zorder>departamento_group</zorder>
  <zorder>txt_asunto</zorder>
  <zorder>btn_cancelar</zorder>
  <zorder>btn_guardar</zorder>
  <zorder>txt_tituloEvento</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>chk_isPrivado</zorder>
 </widget>
 <resources>
  <include location="../terra.qrc"/>
 </resources>
 <connections/>
</ui>
